review/developer/pr-descriptions.md

```diff
@@ -124,17 +124,17 @@ The first sentence describes what's actually being done. The rest of the
 description explains *why* the change is being made and gives the reviewer a lot
 of context.
 
-## Generated CL descriptions
+## Generated PR descriptions
 
-Some CLs are generated by tools. Whenever possible, their descriptions should
+Some PRs are generated by tools. Whenever possible, their descriptions should
 also follow the advice here. That is, their first line should be short, focused,
-and stand alone, and the CL description body should include informative details
-that help reviewers and future code searchers understand each CL's effect.
+and stand alone, and the PR description body should include informative details
+that help reviewers and future code searchers understand each PR's effect.
 
-## Review the description before submitting the CL
+## Review the description before merging the PR
 
-CLs can undergo significant change during review. It can be worthwhile to review
-a CL description before submitting the CL, to ensure that the description still
-reflects what the CL does.
+PRs can undergo significant change during review. It can be worthwhile to review
+a PR description before merging the PR, to ensure that the description still
+reflects what the PR does.
 
-Next: [Small CLs](small-cls.md)
+Next: [Small PRs](small-prs.md)
```

Comment:
Should we add a section before this that clarifies what are the merging strategies we're supporting?

Reply:
@oestoer We definitely should, do you have something in mind?

Reply:
yes, I'll create a separate PR for that